src/content/docs/business-observability/troubleshoot-pathpoint.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Troubleshoot Pathpoint using special filters
+tags:
+  - Business observability
+  - Pathpoint
+  - Troubleshoot Pathpoint
+metaDescription: How to troubelshoot a pathpoint using special filters i.e. Canary, Flame filter.
+---
+
+import stagesTypes from 'images/business-observability-screenshot-stages-types.png'
+
+import canaryFilter from 'images/business-observability-screenshot-canary-filter.png'
+
+import activeCanaryFilter from 'images/business-observability-screenshot-active-canary-filter.png'
+
+import pathpointConfig from 'images/business-observability-screenshot-pathpoint-config.png'
+
+import generalConfig from 'images/business-observability-screenshot-pathpoint-general-config.png'
+
+import credentials from 'images/business-observability-screenshot-pathpoint-credentials.png'
+
+import installJob from 'images/business-observability-screenshot-install-job.png'
+
+import activateFlameFilter from 'images/business-observability-screenshot-activate-flame-filter.png'
+
+import activatedFlameFilter from 'images/business-observability-screenshot-activated-flame-filter.png'
+
+import activateDropFilter from 'images/business-observability-screenshot-activate-drop-filter.png'
+
+import activatedDropFilter from 'images/business-observability-screenshot-activated-drop-filter.png'
+
+Pathpoint allows you to execute different troubleshooting tasks using New Relic Synthetic monitors. These tasks can be run in the background allowing you to proactively identify and resolve issues before they have a significant impact on your customer experience. 
+
+On this doc, you learn to trouble your Pathpoint using:
+
+- Canary filter
+- Flame filter
+- Drop filter
+
+## Canary filter
```

Comment:
I'd divide the content here into steps.

Reply:
Do you mean the steps of applying the filter?

Reply:
already did! :)